docs/reference/reference-training/training/api-pytorch-reference.rst

```diff
@@ -8,105 +8,112 @@
 | :doc:`/training/apis-howto/api-pytorch-ug` |
 +--------------------------------------------+
 
```

Comment:
Might be nice to include a quick summary of how the different headers in this document relate to each other.  Something like:

```
Determined offers a PyTorch-based training loop that is fully integrated with the
Determined platform.  There are two parts:
   -  :class:`~determined.pytorch.PyTorchTrial`, which you must subclass
      to define things like model architecture, optimizer, data loaders, and how to
      train or validate a single batch, and
   -  :class:`~determined.pytorch.Trainer`, which is used for customizing and executing
      the training loop around a ``PyTorchTrial``.
```